### ISSUE TITLE
Wiring help on atmega32u4 3-15v.

### ISSUE THREAD

**Issue by Artasdmc** [Legion2/CorsairLightingProtocol#367]
Does anyone have a picture for me to use as a sample for wiring?

So leds dont get power from the board itself, right?

Signal "guide" which isn't really a guide tells that you need 2 5v inputs, but I've seen a video where the guy uses only one 5v line to power his leds.

I will have it connected/powered through usb header on the motherboard, the 5v line should only power the leds? Or the controller needs 5v line connected to it as well?

**Comment by Legion2** [comment]
You need one 5v power line for your leds (depending on the leds type), but the power supply must be able to provide enough power for all your leds.

**Comment by Artasdmc** [comment]
Cheers, I think one sata 5v line should be enough for a strip that I have in front of the case making a ring around it, since usually the brightness sweetspot for me is about 96 on the strips that i have.

I had CODI6 and I fried my PSU with my other strips going around my table lmao. As they were powered by external 5v 100w supply and I converted strips to use standard argb connectors and connected them to codi6  argb headers (they were still powered by external power supply at the end of them)

So fair to say I was extremely dumb. Current was flowing to my PSU through the sata connection that CODI6 was powered. It was all good and fine when PC powered on, but as soon as it switched off it started frying the psu.
